src/component/time-active/TimeActive.jsx

```diff
@@ -0,0 +1,164 @@
+    if (this.props.activeTimeEvent.id !== nextProps.activeTimeEvent.id) {
+      clearInterval(this.interval);
+      if (nextProps.activeTimeEvent.stop_time == null) {
+        this.interval = setInterval(() => this.updateTime(), 1000);
+      }
+    } else if (nextProps.activeTimeEvent.stop_time != null) {
+      clearInterval(this.interval);
+    }
+    return true;
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  render() {
+    const { classes, handleStartStopClick } = this.props;
+    const currentDay = new Date().getDate();
+    const currentMonth = (new Date().getMonth()) + 1;
+
+    const {
+      skills,
+      templateName,
+      organization_name: organizationName,
+      project_name: projectName,
+      activity,
+      template_id: templateId,
+      stop_time: stopTime,
+    } = this.props.activeTimeEvent;
+
+    const buttonIcon = (stopTime == null) ? <Stop /> : <PlayArrow />;
+    return (
+      <div>
+        <Fragment>
+          <Typography
+            className={classes.currentlyActiveTimeEvent}
+            variant="h5"
+          >
+            {TemplateListConst.CURRENTLY_ACTIVE_TIME_EVENT}
+          </Typography>
+        </Fragment>
+        <Card className={classes.timeActiveTemplate}>
+          <PageSplit
+            left={(
+              <div>
+                <Typography variant="subtitle1">
+                  {'Day'}
+                </Typography>
+                <Typography variant="h4">
+                  {currentDay}
+                </Typography>
+              </div>
+            )}
+            leftSize={6}
+            right={(
+              <div>
+                <Typography variant="subtitle1">
+                  {'Month'}
+                </Typography>
+                <Typography variant="h4">
+                  {currentMonth}
+                </Typography>
+              </div>
+            )}
+            rightSize={6}
+          />
+        </Card>
+
+        <Card className={classes.timeActiveTemplate}>
+          <PageSplit
+            left={(
+              <div>
+                <Typography variant="h4">
+                  {templateName}
+                </Typography>
+                <Typography className={classes.organizationAndProject} variant="h5">
+                  {`${organizationName} > ${projectName}`}
+                </Typography>
+                <Typography className={classes.activity} variant="body1">
+                  {`${TemplateConst.ACTIVITY}: ${activity}`}
+                </Typography>
+                <Typography variant="h3">
+                  {this.formatDuration(this.props.activeTimeEvent.duration)}
+                </Typography>
+                <Typography className={classes.skills} variant="body1">
+                  {`${TemplateConst.SKILLS}: ${this.getSkillsList(skills)}`}
+                </Typography>
+                <TextField label="What did you do?" placeholder="Implemented some changes" />
```

Comment:
Please create a constants file for these two strings and use them here

Reply:
Done